knowledge_repo/app/routes/web_editor.py

```diff
@@ -1,13 +1,11 @@
-from builtins import range
-from builtins import str
 import json
 import logging
 import sys
 import os
-from collections import defaultdict
+from builtins import str
```

Comment:
This is not a good idea, since it seems to break string handling in places outside of our project (i.e. not all packages seem to support the unicode strings that result...). That's been my experience anyway.